pxf/pxf-json/src/main/java/org/apache/hawq/pxf/plugins/json/JsonAccessor.java

```diff
@@ -0,0 +1,66 @@
+package org.apache.hawq.pxf.plugins.json;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import java.io.IOException;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.hadoop.mapred.FileSplit;
+import org.apache.hadoop.mapred.InputSplit;
+import org.apache.hadoop.mapred.JobConf;
+import org.apache.hawq.pxf.api.utilities.InputData;
+import org.apache.hawq.pxf.plugins.hdfs.HdfsSplittableDataAccessor;
+
+/**
+ * This JSON accessor for PXF will read JSON data and pass it to a {@link JsonResolver}.
+ * 
+ * This accessor supports a single JSON record per line, or a more "pretty print" format.
+ */
+public class JsonAccessor extends HdfsSplittableDataAccessor {
+
+	public static final String IDENTIFIER_PARAM = "IDENTIFIER";
+	public static final String ONERECORDPERLINE_PARAM = "ONERECORDPERLINE";
+
+	private String identifier = "";
+	private boolean oneRecordPerLine = true;
+
+	public JsonAccessor(InputData inputData) throws Exception {
+		super(inputData, new JsonInputFormat());
+
+		if (!StringUtils.isEmpty(inputData.getUserProperty(IDENTIFIER_PARAM))) {
+			identifier = inputData.getUserProperty(IDENTIFIER_PARAM);
```

Comment:
JsonAccessor supports two modes: 
1. json object per line -> LineRecordReader
2. multiline json objects that can span beyond split's boundries. -> JsonRecordReader + JsonStreamReader
Second mode (used to be called JSON-PP e.g. Pretty Print) requires an json element name (e.g. identifier) to be set to identify the boundries of a json record. 
I've removed the JSON-PP instead the Json profile without IDENTIFIER set will default to mode 1 (e.g. json per line) and if the identifier paramter is set switches to mode 2 (multiline json records named by the identifer param).
Will update the documentation to reflect those changes.